plugins/dell-dock/dell-dock.quirk

```diff
@@ -126,15 +129,32 @@ DellDockHubVersionLowest = 1.31
 DellDockBlobMajorOffset = 0x400a
 DellDockBlobMinorOffset = 0x4009
 
-# USB4 controller
+# USB4 controller (TBT)
 [TBT-00d4b071]
-Name = USB4 controller in Dell dock
+Name = USB4 controller (TBT) in Dell dock
 Summary = USB4 controller
 Vendor = Dell Inc.
 VendorId = TBT:0x00D4
 ParentGuid = USB\VID_413C&PID_B06E&hub&embedded
 FirmwareSizeMin=0x40000
 FirmwareSizeMax=0x80000
-Flags = skips-restart,require-ac,dual-image
+Flags = skips-restart,require-ac,dual-image,updatable
 Icon = thunderbolt
 InstallDuration = 46
+
+# USB4 controller (USB)
+[USB\VID_8087&PID_0B40]
+Name = USB4 controller (USB) in Dell dock
+Summary = USB4 controller
+Vendor = Intel Inc.
+VendorId = USB:0x8087
```

Comment:
I would think that you want override these to Dell or preferably remove this entry all-together and re-use the other entry from thunderbolt plugin

Reply:
It's preferably overridden with Dell Vendor but I wonder if confusion arise from the Intel one in reality.